client/js/views/GameView.vue

```diff
@@ -709,9 +709,8 @@ export default {
   watch: {
     logs: function () {
       this.$nextTick(function () {
-        const container = this.$refs.logsContainer;
-        if (container) {
-          container.scrollTop = container.scrollHeight + 120;
+        if (this.$refs.logsContainer) {
```

Comment:
Let's put this conditional inside the function so it's called whenever the function is called

Reply:
ok good idea

Reply:
I moved the conditional inside of the function